client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button, CircularProgress, InputLabel } from '@mui/material';
+import { Box } from '@mui/system';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import { makeStyles } from '@mui/styles';
+import postPhoto from '../../../helpers/APICalls/postPhoto';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import { useState } from 'react';
+
+import editProfilePhotoUri from '../../../helpers/APICalls/editProfilePhotoUri';
+
+const useStyles = makeStyles({
+  dateInput: {
+    borderRadius: 8,
+    border: '1px solid #dbdbdb',
+    fontSize: 16,
+    width: '100%',
+    padding: '15px',
+  },
+});
+
+interface ProfilePhotoProps {
+  header: string;
+  currentUser?: User; // set to optional but always passed in from settings
+  currentProfile?: any;
+}
+
+const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, currentUser, currentProfile }) => {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  const [file, setFile] = useState();
+  const [images, setImages] = useState([]);
+  const [isSubmitting, setSubmitting] = useState(false);
+
+  console.log(currentProfile);
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!

Reply:
Understood, addressed.